LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentClient.java

```diff
@@ -27,6 +27,10 @@ public class LocalPaymentClient {
     private final LocalPaymentApi localPaymentApi;
     private LocalPaymentListener listener;
 
+    /// Used for linking events from the client to server side request
+    /// In the Local Payment flow this will be a Payment Token/Order ID
```

Comment:
I think the triple slash might be an iOS pattern. We can use Kdoc syntax here:

```kotlin
/**
 * Used for linking events from the client to server side request in the Local Payment flow this 
 * will be a Payment Token/Order ID
 */
```

https://kotlinlang.org/docs/kotlin-doc.html

Reply:
knowledge shared = knowledge²
Addressed on 6110eb13f9023fb24dfe6d448cc2068968662db3